addons/common/fnc_isTurnedOut.sqf

```diff
@@ -27,54 +27,53 @@ Examples:
 
 Author:
     Nou, courtesy of ACRE project.
+    shukari
 ---------------------------------------------------------------------------- */
 SCRIPT(isTurnedOut);
 
 params ["_unit"];
 
-private _turn = false;
-private _out = false;
+if (isNull objectParent _unit) exitWith {true};
+if (isTurnedOut _unit) exitWith {true};
+
+private _return = false;
 private _vehicle = vehicle _unit;
-private _cfg = configFile >> "CfgVehicles" >> (typeOf _vehicle);
-
-if (_vehicle != _unit) then {
-    if (driver _vehicle == _unit) then {
-        private _forceHideDriver = getNumber(_cfg >> "forceHideDriver");
-        _turn = (_forceHideDriver != 1);
-        _out = (_vehicle animationPhase "hatchDriver") > 0;
-    } else {
-        private _assignedRole = assignedVehicleRole _unit;
-        if (_assignedRole select 0 == "Turret") then {
-            private _turretPath = _assignedRole select 1;
-            private _turret = [_vehicle, _turretPath] call CBA_fnc_getTurret;
-
-            private _canHideGunner = getNumber(_turret >> "canHideGunner");
-            private _forceHideGunner = getNumber(_turret >> "forceHideGunner");
-            _turn = (_canHideGunner == 1 && _forceHideGunner == 0);
-
-            private _hatchAnimation = getText(_turret >> "animationSourceHatch");
-            _out = (_vehicle animationPhase _hatchAnimation) > 0;
-
-        } else {
-            if ((_assignedRole select 0) == "Cargo") then {
-                private _attenuateCargo = getArray(_cfg >> "soundAttenuationCargo");
-                if ((count _attenuateCargo) > 0) then {
-                    private _index = (count _attenuateCargo)-1; // wait for command to get cargo index
-
-                    if (_index > -1) then {
-                        if(_index > (count _attenuateCargo)-1) then {
-                            _index = (count _attenuateCargo)-1;
-                        };
-                        if((_attenuateCargo select _index) == 0) then {
-                            _out = true;
-                        };
-                    };
-                };
-            };
-        };
+private _role = _unit call CBA_fnc_vehicleRole;
+
+// FFVs are allways outside
+if ([_unit] call ace_common_fnc_getTurretIndex in ([_vehicle] call ace_common_fnc_getTurretsFFV)) then {_return = true};
+
+// Spezial condition for RHS car windows
+if (_vehicle isKindOf "rhsusf_m1025_w" && {_role in ["driver", "Turret"]}) then {
```

Comment:
hack for 3rd party addons?

Reply:
Yes, extra with a comment... is nothing for CBA but for other mods i think... i would like to make it possible to add extra stuff for addons to make it more variable...
This file is atm heavy at WIP and i try all to make it working with all i can test with my available mods.

Reply:
Maybe replace it with some config values check and then ask RHS to add them to their source.

Reply:
Config is not possible for that... this are the windows that you can open via a useraction.
So its highly variable... i dont now how... i suppose a eventhandler thing or something is needed here.
It would be nice if this function can be changed by outside so mission/mod makers can make there own things and this function only make the static things.

Reply:
just checked and it looks like all `Turret` roles in these RHS Humvees are FFV so and no sense to check windows because FFV are checked above

Reply:
because if the window is closed it returns false